### PR TITLE
Added back button on Student List page

### PR DESCRIPTION
**Pull request by monaviaco**

### CODE REVIEW

File: classroom-dashboard/app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -81,7 +79,10 @@ div.style_alien_navbar img {
     margin-right: auto;
 }
 
-/*.new_student_link{
+/*
+Are we still using these links?
```
Comment (diarreola) [inline classroom-dashboard/app/assets/stylesheets/application.css]: this code is not utilized within our application, so we should probably delete it.

File: classroom-dashboard/app/controllers/sections_controller.rb [hunk 2]
```diff
@@ -16,6 +16,7 @@ def new
 		@section = Section.new
 	end
 
+	# Does this edit function work by default?
```
Comment (diarreola) [inline classroom-dashboard/app/controllers/sections_controller.rb]: thanks for catching this!
I believe it should have the following: 
@student =Student.find(params[:id])

Try running it with the above. If it doesn't work, we know theres another bug in our code.

File: classroom-dashboard/app/views/students/index.html.erb [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
-  <div style="text-align:center">
-    <p><%= link_to "Add Student", new_section_student_path(@section)%></p>
-  </div>
+<h5><%= link_to "Back to Course", section_path(@section)%></h5>
 
+<div style="text-align:center">
+  <p><%= link_to "Add Student", new_section_student_path(@section)%></p>
+</div>
+
+<!-- Are these still needed? -->
```
Comment (diarreola) [inline classroom-dashboard/app/views/students/index.html.erb]: <br>produces a line break in text. It doesn't seem like its being utilized the way we implemented it. I would say delete :)

File: classroom-dashboard/app/views/students/show.html.erb [hunk 4]
```diff
@@ -1,17 +1,19 @@
 <div style="text-align:center">
 <!-- individual student's page -->
```
Comment (diarreola) [inline classroom-dashboard/app/views/students/show.html.erb:2]: thanks for adding the comment!

**Review by monaviaco — COMMENTED** [review]
Addressed all comments, deleted the unnecessary/old code and updated the edit function for students!

**Review by diarreola — APPROVED** [review]

**Pull request merged**